### PR TITLE
[Bots] Signals Intelligence

### PR DESCRIPTION
**Pull request by patriciasantaana**
PCX-9658 / PCX-10168

### CODE REVIEW

File: content/bots/concepts/signals-intelligence.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+pcx_content_type: concept
+title: Signals Intelligence
+---
+
+# Bots Signals Intelligence
+
+For every available [JA4 Fingerprint](/bots/concepts/ja3-ja4-fingerprint/), Bot Management customers can view how Cloudflare sees it on the internet and what behavior we view with the fingerprint. This data can help you understand why a request is scored in a particular fashion or allow you to use the aggregate data in your own ML models, run in either [Cloudflare Workers](/workers/) or at the origin location.
```
Comment (patriciasantaana) [inline content/bots/concepts/signals-intelligence.md]: Compiles is failing due to this broken link but it is dependent on another PR (#13416). Should be fine overall.

Comment (maxvp) [inline content/bots/concepts/signals-intelligence.md]:
```suggestion
For every available [JA4 Fingerprint](/bots/concepts/ja3-ja4-fingerprint/), Bot Management customers can view how Cloudflare sees it on the Internet and what behavior we view with the fingerprint. This data can help you understand why a request is scored in a particular fashion or allow you to use the aggregate data in your own ML models, run in either [Cloudflare Workers](/workers/) or at the origin location. 
```

File: content/bots/concepts/signals-intelligence.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+# Bots Signals Intelligence
+
+For every available [JA4 Fingerprint](/bots/concepts/ja3-ja4-fingerprint/), Bot Management customers can view how Cloudflare sees it on the internet and what behavior we view with the fingerprint. This data can help you understand why a request is scored in a particular fashion or allow you to use the aggregate data in your own ML models, run in either [Cloudflare Workers](/workers/) or at the origin location. 
+
+Specifically, for each JA4 fingerprint, you will be able to access the following information: 
+
+- The percentage of traffic Cloudflare sees is associated with browsers. 
+- The percentage of traffic Cloudflare sees is associated with known bots.
```
Comment (maxvp) [inline content/bots/concepts/signals-intelligence.md]: I may be misreading these, but I don't think these make sense. Maybe `The percentage of traffic associated with browsers that Cloudflare sees`?

**Review by maxvp — APPROVED** [review]

File: content/bots/concepts/signals-intelligence.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+pcx_content_type: concept
+title: Signals Intelligence
+---
+
+# Bots Signals Intelligence
+
+For every available [JA4 Fingerprint](/bots/concepts/ja3-ja4-fingerprint/), Bot Management customers can view how Cloudflare sees it on the Internet and what behavior we view with the fingerprint. This data can help you understand why a request is scored in a particular fashion or allow you to use the aggregate data in your own ML models, run in either [Cloudflare Workers](/workers/) or at the origin location.
```
Comment (patriciasantaana) [inline content/bots/concepts/signals-intelligence.md:8]:
```suggestion
For every available [JA4 Fingerprint](/bots/concepts/ja3-ja4-fingerprint/), Bot Management customers can view how Cloudflare sees it on the Internet and what behavior we view with the fingerprint. This data can help you understand why a request is scored in a particular fashion or allow you to use the aggregate data in your own ML models, run in either [Cloudflare Workers](/workers/) or at the origin location. 
```

**Pull request merged**